ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -243,17 +243,11 @@ function getDiseaseNames(keyword) {
     .then((response) => {
       return response.json();
     })
-    .then((matches) => {
-      const labeledMatches = fp.filter((elt) => elt.hasOwnProperty('label'))(
-        matches
-      );
-      // Property 'label' does not exist on type 'unknown'
-      // TODO RW-5572 confirm proper behavior and fix
-      // eslint-disable-next-line @typescript-eslint/dot-notation
-      const diseases = fp.map((elt) => elt['label'])(labeledMatches);
```

Comment:
looks like the linter didn't understand the `fp` version, but the non-fp version works.